### PR TITLE
Updated license

### PR DESCRIPTION
**Pull request by mcollina**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,10 @@
     "json"
   ],
   "author": "Matteo Collina <hello@matteocollina.com>",
+  "contributors": [
+    "David Mark Clements <huperekchuno@googlemail.com>",
```
Comment (davidmarkclements) [inline package.json]: should this actually be nearform addr?

Comment (davidmarkclements) [inline package.json]: (same q. for you)

Comment (mcollina) [inline package.json]: not really.

Comment (marcbachmann) [inline package.json]: please use the email you're using for the commits

**Comment by mcollina** [comment]
@davidmarkclements are you good to go?

**Comment by davidmarkclements** [comment]
yep

**Pull request merged**